library/swift/ios/UserDefaultsStore.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// Simple implementation of a `KeyValueStore` leveraging `UserDefaults` for persistence.
+@objcMembers
+public class UserDefaultsStore: KeyValueStore {
+  private let defaults: UserDefaults
+
+  public init(userDefaults: UserDefaults) {
+    self.defaults = userDefaults
+  }
+
+  public func readValue(forKey key: String) -> String? {
+    return defaults.string(forKey: key)
```

Comment:
I think that in general - even though it's not codified anywhere - we used to write `self.` when referring to instance variables. Not a blocker but consider adding `self.` for consistency.